File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcResource/IfcConstructionResource/IfcConstructionResource.vb
Public Class IfcConstructionResource Inherits IfcResource
    Public Usage As IfcResourceTime
    Public BaseCosts As List(Of IfcAppliedValue)
    Public BaseQuantity As IfcPhysicalQuantity

    ' === EXTENDED BY ===
    ' IfcConstructionEquipmentResource
    ' IfcConstructionMaterialResource
    ' IfcConstructionProductResource
    ' IfcCrewResource
    ' IfcLaborResource
    ' IfcSubContractResource
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcResource/IfcConstructionResource/IfcConstructionResource.cs
public class IfcConstructionResource : IfcResource
{
    public IfcResourceTime Usage { get; set; }
    public List<IfcAppliedValue> BaseCosts { get; set; }
    public IfcPhysicalQuantity BaseQuantity { get; set; }

    // === EXTENDED BY ===
    // IfcConstructionEquipmentResource
    // IfcConstructionMaterialResource
    // IfcConstructionProductResource
    // IfcCrewResource
    // IfcLaborResource
    // IfcSubContractResource
}
